README.md

```diff
@@ -31,6 +31,60 @@ bin/rake
 
 *Note: As uploads use S3, you'll need to add details to the .env file for them to work. Alternatively, [replace S3 with ActiveStorage](https://github.com/forsbergplustwo/partner-metrics/issues/20).*
 
+## S3 - Minimum setup guide
+
+### 1. Sign up for or login to [Amazon AWS](https://aws.amazon.com)
+### 2. Create an S3 Bucket:
+* In the AWS console, search for S3 and click the offering in the drop down menu. 
+* In the S3 Management Console, click the "Create Bucket" button. 
+* In the bucket creation wizard - name it something cool. (eg... "ice-bucket")
+* Take note of the region your bucket is in. In my case I used the default presented ("us-east-2").
+* Leave all other default settings alone. Click create to finalize the creation of the bucket. 
+
+### 2b. Allow CORS
+* While looking at the individual bucket overview of the bucket you just created, click into the "Permissions" tab. Scroll down to "Cross-origin resource sharing (CORS)" and paste in the following:
+```
+[
+    {
+        "AllowedHeaders": [
+            "*"
+        ],
+        "AllowedMethods": [
+            "GET",
+            "HEAD",
+            "POST",
+            "PUT",
+            "DELETE"
```

Comment:
We don't need `DELETE` access, so may be good to remove. The rest looks fine for development, but maybe we can add some text after this code block saying:

_Note: Make sure to update the `AllowedOrigins` details above with your production app details, if your app is being made public._